website/siteConfig.js

```diff
@@ -52,8 +45,76 @@ const users = [
     fbOpenSource: false,
     pinned: false,
   },
+  {
+    caption: "Storybook",
+    image: "img/users/storybook.svg",
+    infoLink: "https://storybook.js.org",
+    fbOpenSource: false,
+    pinned: false,
+  },
+  {
+    caption: "create-react-app",
+    image: "img/users/create-react-app.svg",
+    infoLink: "https://facebook.github.io/create-react-app/",
+    fbOpenSource: true,
+    pinned: false,
+  },
+  {
+    caption: "Gatsby",
+    image: "img/users/gatsby.svg",
+    infoLink: "https://www.gatsbyjs.org/",
+    fbOpenSource: false,
+    pinned: false,
+  },
+  {
+    caption: "Uppy",
+    image: "img/users/uppy.svg",
+    infoLink: "https://uppy.io/",
+    fbOpenSource: false,
+    pinned: false,
+  },
+  {
+    caption: "Aurelia",
+    image: "img/users/aurelia.svg",
+    infoLink: "https://aurelia.io/",
+    fbOpenSource: false,
+    pinned: false,
+  },
+  {
+    caption: "Bit",
+    image: "img/users/bit.png",
+    infoLink: "https://bit.dev/",
+    fbOpenSource: false,
+    pinned: false,
+  },
+  {
+    caption: "Yoshi",
+    image: "img/users/yoshi.webp",
+    infoLink: "https://wix.github.io/yoshi/",
+    fbOpenSource: false,
+    pinned: false,
+  },
+  {
+    caption: "AWS CDK",
+    image: "img/users/aws-cdk.png",
+    infoLink: "https://docs.aws.amazon.com/cdk/api/latest/",
+    fbOpenSource: false,
+    pinned: false,
+  },
+];
+
+const companyUsers = [
+  {
+    caption: "SheetJS",
```

Comment:
SheetJS is an open source project.

Reply:
Should I make a distinction between sponsors and other open source projects then?🙂